manager/system_handler.py

```diff
@@ -412,26 +413,34 @@ def handle_get(cls, **kwargs):  # pylint: disable=too-many-statements,too-many-b
             raise ApplicationException('inventory_id exists but is not evaluated', 404)
 
         rh_account_id = get_account_id(connexion.context['user'])
-        cves_view = SystemCvesView(rh_account_id, list_arguments, {'inventory_id': inventory_id}, args, args, connexion.request.path, cls._ids_only)
 
+        cve_advisories = {}
+        patch_access = None
+        if 'show_advisories' in args and args['show_advisories']:
+            try:
+                raw_cve_names = (SystemVulnerabilities.select(CveMetadata.cve.alias('cve_name'), )
+                                 .join(SystemPlatform, on=(SystemVulnerabilities.system_id == SystemPlatform.id))
+                                 .join(CveMetadata, on=(SystemVulnerabilities.cve_id == CveMetadata.id))
+                                 .join(CveAccountData, JOIN.LEFT_OUTER, on=((CveAccountData.cve_id == CveMetadata.id)
+                                                                            & (CveAccountData.rh_account_id == rh_account_id)))
+                                 .where(SystemVulnerabilities.rh_account_id == rh_account_id)
+                                 .where(SystemPlatform.inventory_id == inventory_id)).dicts()
+            except DoesNotExist as exc:
+                raise ApplicationException('inventory_id must exist and inventory_id must be visible to user', 404) from exc
+            cve_names = [cve['cve_name'] for cve in raw_cve_names]
+            # if show_advisories parameter is present - get advisory data from PATCH
+            cve_advisories, patch_access = cls._get_advisories_per_cve(cve_names, inventory_id)
```

Comment:
Try to apply same optimizations as mentioned in cve_handler.py.

Reply:
Done